rbac/rbac/settings.py

```diff
@@ -394,31 +394,40 @@
     NOTIFICATIONS_TOPIC = None
 
 # Kafka settings
+KAFKA_SERVERS = []
+
 if KAFKA_ENABLED:
     KAFKA_AUTH = {}
     if ENVIRONMENT.bool("CLOWDER_ENABLED", default=False):
-        kafka_broker = LoadedConfig.kafka.brokers[0]
-        KAFKA_HOST = kafka_broker.hostname
-        KAFKA_PORT = kafka_broker.port
+        kafka_brokers = LoadedConfig.kafka.brokers
+        if not kafka_brokers:
+            raise ValueError("No kafka brokers available")
+        for broker in kafka_brokers:
+            kafka_host = broker.hostname
+            kafka_port = broker.port
+            kafka_info = f"{kafka_host}:{kafka_port}"
+            KAFKA_SERVERS.append(kafka_info)
+
         try:
-            if kafka_broker.authtype.value == "sasl":
+            if kafka_brokers[0].authtype.value == "sasl":
                 KAFKA_AUTH.update(
                     {
-                        "bootstrap_servers": f"{KAFKA_HOST}:{KAFKA_PORT}",
-                        "sasl_plain_username": kafka_broker.sasl.username,
-                        "sasl_plain_password": kafka_broker.sasl.password,
-                        "sasl_mechanism": kafka_broker.sasl.saslMechanism.upper(),
-                        "security_protocol": kafka_broker.sasl.securityProtocol.upper(),
+                        "bootstrap_servers": KAFKA_SERVERS,
```

Comment:
not sure if I understand what is expected here? here we are using only the first broker but we set bootstrap servers param to all brokers?

Reply:
From what I understand,  the  kafka producer can only have one type of authentication. So  if there is one broker that has that information, then KAFKA_AUTH can use that information (like the sasl_plain_password, sasl_mechanism and security protocol), and the other fields, while the bootstrap servers can take in a list of servers, which is why we have the kafka_servers variable there! 
Which is why I was thinking that if we can just the first one, but if the first broker doesn't have it, but the third broker has that information...hmmm I might have to make a different variable then...

Reply:
Here is what the brokers look like in terms of the authentication for rerference: 
![Screenshot from 2024-02-21 08-10-54](https://github.com/RedHatInsights/insights-rbac/assets/29558356/5018b152-db5a-4533-964d-da3d0f6e4906)

Reply:
yes that is what is not clear to me, why we set KAFKA_AUTH from first broker and not from e.g. second ... but I dont have knowledge or preference here

Reply:
Oh! All of the kafka_auth are the same for each of the brokers! If you look at the screenshot, you can see how the fields are the same for each one (and the data for each one is the same)  so that's why if we set from KAFKA_AUTH from the first broker, it should work for the other 2 brokers! If it helps, I'm also down to call/meet/talk about it anytime too!!

Reply:
I see 🤩 that's the part I haven't seen before!!! thank you Ellen for the explanation